src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
Spring Actuator should be configured here to prevent unauthorised use of management endpoints.